content/livesync/models.textile

```diff
@@ -1,82 +1,160 @@
+
+```[javascript]
+const modelsClient = new ModelsClient({
+ ably,
+ logLevel,
+ syncOptions: {
+  historyPageSize,
+  messageRetentionPeriod,
+  retryStrategy,
+ },
+ eventBufferOptions: {
+  bufferMs,
+  eventOrderer,
+ },
+ optimisticEventOptions: {
+  timeout,
+ },
+});
+```
+
+h2(#create-or-retrieve). Create or retrieve a model
+
+Instantiate a model using a unique channel name using the @models.get()@ method on the models client. A unique channel name identifies the model on the client and corresponds to the "channel":/channels name used to subscribe to state updates from the backend. If the channel name already exists, then the existing channel will be returned:
 
 ```[javascript]
 const model = modelsClient.models.get({
-  channelName: 'post:123',
+  channelName: 'posts:123',
   sync,
   merge,
 });
 ```
 
-h2(#sync). Sync functions
+TODO: Diagram of how the model components work together
+
+h2(#designing). Designing your Channels and Models
 
-The sync function instructs your model to initialize with the latest data from the backend and is necessary when creating a model. It can be any function that optionally accepts parameters and returns a promise with the latest state of your data model, along with a "@sequenceId@":#sequence.
+In general, when designing your channels and models, aim for a 1:1:1 relationship between channels, models, and your domain model objects. Channels facilitate data partitioning and access control. For instance, in a task management app, each task can be represented by a model with its own channel, ensuring updates are specific to each task. Ably's "Token Capabilities":/auth/capabilities are applied on a per channel basis allowing fine-grained access control for each data model in your application.
 
-The following is an example sync function that fetches the latest model state from your database:
+For infrequently updated models, grouping them on the same channel may be viable, while frequently changing models benefit from individual channels. Starting with one channel per model and one model per domain object is a good baseline, adjusting as needed.
+
+h2(#create-sync). Create a Sync function
+
+The sync function is one of the core functions required for the Models SDK to function. This function is called when the model is initialized to retrieve the latest data from your backend.
+
+The function can contain any custom functionality within it, but needs to return a promise with two objects within the response. The first object is a field called a "@sequenceId@":#sequence-id, which is used to identify the point in the stream of change events that corresponds to the current version of the database's state.
+
+The following is an example sync function that fetches the latest model state from your database by calling the @/api/posts@ endpoint in your backend:
 
 ```[javascript]
-async function sync(id: number, page: number) {
```

Comment:
I think the textile style code highlights in the code snippets comments are confusing. Perhaps use back ticks or double quotes?